tests/mime/mime-dec-parse-full-msg-test01/README.md

```diff
@@ -0,0 +1,11 @@
+# Test Description
+
+Test some mimre processing
+
+## PCAP
+
+Previous unit test for MIME in Suricata
```

Comment:
They all say this but have a pcap within every test. Can we deduplicate it?
createst cannot do this at the moment. It copies pcap files to the test..

Reply:
These are different unit tests / pcaps...
Do I miss something ?

Reply:
I'm assuming she thought they were all the same... 🤔  maybe just some rewording.
Maybe: `Adapted using data from a previous specific unit test for MIME in Suricata.` ?

Reply:
ok